### DISCUSSION TITLE
Persistent mode for KeyClu

### DISCUSSION THREAD

**Discussion by hericius** [Anze/KeyCluCask#14] [Ideas]
Hello,

Could you consider an option that would allow you to keep the current window open so that the shortcuts are always displayed?

Regards,
André,

**Comment by Anze** [comment]
At the moment I do not have plans to implement it. Still, If more people express the need for sticky window, I'll be happy to implement it in the future. Meantime, if you really need this feature right away you can try ShowMeYourHotKeys.

**Comment by hericius** [comment]
Hi,

Thank you for your reply.

The sticky window doesn't need to be urgently available, I can do without it.

;-)


Sincères salutations | Met vriendelijke groeten | Kind regards

**Comment by Anze** [comment]
Hey, i've made a persistent panel. It's a basic one at the moment, but will be improved in future.

**Comment by djpowers** [comment]
I appreciate the inclusion of a persistent panel! One improvement I'd like to see is an easier way to close it. The triple-⌘ seems like an appropriate command to show it, but I find myself trying to close it with ⌘-W or Esc which does not work. I think a close button somewhere on the window could also be helpful.

**Comment by Anze** [comment]
Hey [djpowers](https://github.com/djpowers),
The latest release have possibility to keep panel opened (another trigger to dismiss) and can be closed by ⎋ (ESC) key or any other non-modifier key pressed (this also includes hotkey combination).

**Reply by djpowers** [reply]
Nice! Should that work on version 0.20, or is this part of an upcoming release?

**Reply by Anze** [reply]
Oh, i just realized that it was asked for Persistent Panel, so far it is done for Shortcuts Panel. Persistent Panel would be adjusted in upcoming release. Sorry for misleading.

**Reply by djpowers** [reply]
Got it, thanks for clarifying. The changes for the shortcuts panel are nice, and I look forward to the updates for the persistent panel.
